clone_nonbillables_and_process.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+set -xe
+
+# Add deploy key to ssh config
+mkdir -p ~/.ssh
+touch ~/.ssh/config
+touch ~/.ssh/id_nonbillable
+if ! grep -q "Host github-nonbillable" ~/.ssh/id_nonbillable; then
+    echo "
+    Host github-nonbillable
+        HostName github.com
+        IdentityFile ~/.ssh/id_nonbillable
+    " > ~/.ssh/config
```

Comment:
If this is run from someone's machine, rather than a temporary container, it is going to destroy their `.ssh/config` file. Is there a way to do this without changing the ssh config file?

Reply:
good catch, one could do `>> ~/.ssh/config` to append to file instead.

Reply:
Actually, my preference would be to stop execution if it detects the presence of a `~/.ssh/config` file.

If such a file exists, then this is running on someone's machine rather than a temporary container. In which case they probably already have an SSH key that has access to that repository already configured.

Reply:
I changed the if statement to check if `~/.ssh/config` exists instead.